src/backends/glvisualize.jl

```diff
@@ -0,0 +1,90 @@
+
+
+# [WEBSITE]
+
+# ---------------------------------------------------------------------------
+
+immutable GLScreenWrapper
+    window
+    render
+end
+
+function _create_plot(pkg::GLVisualizePackage; kw...)
+  d = Dict(kw)
+  # TODO: create the window/canvas/context that is the plot within the backend (call it `o`)
+  # TODO: initialize the plot... title, xlabel, bgcolor, etc
+  w,r=GLVisualize.glscreen()
+  @async r()
+  Plot(GLScreenWrapper(w,r), pkg, 0, d, Dict[])
+end
+
+
+function _add_series(::GLVisualizePackage, plt::Plot; kw...)
+  d = Dict(kw)
+  # TODO: add one series to the underlying package
+  push!(plt.seriesargs, d)
+  x,y,z=map(Float32,d[:x]), map(Float32,d[:y]), map(Float32,d[:surface].surf)
+  GLVisualize.view(GLVisualize.visualize(x*ones(y)', ones(x)*y', z, :surface))
```

Comment:
I'm assuming that this both updates **and** displays what is on screen?  If so, you probably want to move the call to `GLVisualize.view` to the `display` method below.

`_create_plot` is for initializing the internals, `_add_series` is for updating the internals for a new data series, and the `writemime`/`display` methods below are for taking the internal representation and saving, displaying, or whatever else.  Ideally it's possible to be able to do this:

```
using Plots; glvisualize()
plot(rand(10));

# no windows have been created or displayed

... do other stuff ...

# now display it
gui()
```

I really should learn the GLVisualize internals...